appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/view-and-review.controller.js

```diff
@@ -6,50 +6,45 @@ import {
 } from './view-and-review.mapper.js';
 import { patchInterestedPartyCommentStatus } from './view-and-review.service.js';
 
+/** @typedef {import("../../appeal-details.types.js").WebAppeal} Appeal */
+/** @typedef {import("../interested-party-comments.types.js").Representation} Representation */
+
 /**
  *
- * @param {import('@pins/express/types/express.js').Request} request
- * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ * @param {(appealDetails: Appeal, comment: Representation) => PageContent} contentMapper
+ * @param {string} templatePath
+ * @returns {import('@pins/express').RenderHandler<unknown>}
  */
-export async function renderViewInterestedPartyComment(request, response) {
-	const { errors, currentComment } = request;
+const render = (contentMapper, templatePath) => (request, response) => {
```

Comment:
Feel free to poopoo this if it seems overcomplicated, it just seemed like an obvious factorisation to me

Reply:
This is fine in principle, my only concern is we're not using this pattern anywhere else. It might just be that there's no use case for it anywhere else though. Either way, happy for this to stay in.

Reply:
I'm pretty sure this specific expression only exists here, because 404ing if there's no comment data doesn't really make sense anywhere else. I wouldn't really call this a patter, it's just writing `zx + yx` as `(z + y)x`